src/main/webapp/js/finscholar/homepagecontroller/homepagecontroller.js

```diff
@@ -16,54 +16,112 @@
 
 goog.module('finscholar.homepagecontroller');
 
+const GoogDom = goog.require('goog.dom');
```

Comment:
the style for 'goog.dom' is to be 'const googDom', same for all closure imports that are lowercase.